api/wait.py

```diff
@@ -1,25 +1,37 @@
-import requests
-import urllib.parse
 import json
+import urllib.parse
+from typing import TYPE_CHECKING
 
+import requests
 
-def wait(update, context):
+if TYPE_CHECKING:
+    import telegram
+    import telegram.ext
+
+
+def wait(update: 'telegram.Update', context: 'telegram.ext.CallbackContext') -> None:
     """What Anime Is This?"""
+    if update.message:
+        message: 'telegram.Message' = update.message
+    else:
+        return
```

Comment:
why do this if you're reassigning immediately after this

Reply:
`update.message` might not exist, return immediately. Later down we try for `update.reply_to_message` that may or may not exist.

Reply:
so we check for both in the try block, something like
```suggestion
    if update.message and update.message.reply_to_message:
        message: 'telegram.Message' = update.message.reply_to_message
    else:
        return
```